src/dates.js

```diff
@@ -1,3 +1,4 @@
 export { default as formatDate } from './dates/format-date';
 export { default as parseDate } from './dates/parse-date';
-export { default as dateFormatString } from './dates/date-format-string';
\ No newline at end of file
+export { default as dateFormatString } from './dates/date-format-string';
```

Comment:
I won't :) We need to accept the changes here before move to the `kendo-angular-intl` repo anyway